example/lib/component/password_field.dart

```diff
@@ -32,19 +30,11 @@ class _PasswordFieldState extends State<PasswordField> {
       keyboardType: TextInputType.text,
       obscureText: true,
       validator: validatePassword,
-      autovalidate: _isPasswordDirty,
+      autovalidateMode: AutovalidateMode.always,
```

Comment:
I think the equivalent of the `_isPasswordDirty` boolean is `autovalidateMode: AutovalidateMode.onUserInteraction`.

Reply:
You are right. Thanks